build.gradle

```diff
@@ -18,7 +18,7 @@ buildscript {
 
     // The 'CI' build variable will evaluate to 'true' when build script is executed via GitHub Actions
     // Ref: https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables
-    def roomVersion = System.getenv('CI') ? java7SafeRoomVersion : m1MacSafeRoomVersion
+    def roomVersion = System.getenv('CI') && !System.getenv('SECURITY_SCAN') ? java7SafeRoomVersion : m1MacSafeRoomVersion
```

Comment:
Hey @raanand-hw does your security scan have a dependency on a particular version of Jetpack Room?